persistence/proto/src/main/scala/net/liftweb/proto/ProtoUser.scala

```diff
@@ -864,8 +864,10 @@ trait ProtoUser {
       }
     }
 
+    val emailElemId = nextFuncName
+    S.appendJs(Focus(emailElemId))
     val bind =
-      ".email" #> FocusOnLoad(<input type="text" name="username"/>) &
+      ".email" #> <input type="text" name="username" id={emailElemId}/> &
```

Comment:
Yeah, I was thinking just making this:

```scala
      ".email" #> <input type="text" name="username" id={emailElemId} onclick="this.focus()" /> &
```

Or somesuch? Since this will go through the LiftMerge extractor, that
will take care of auto-assigning an `id` if needed, etc, etc.

Though it's also worth mentioning that it feels like we shouldn't be
replacing the whole `.email` element at all, but rather just binding
its `name` and possibly its `onclick`:

```scala
".email [name]" #> "username" &
".email [onclick]" #> "this.focus()" &
".password [name]" #> "password"
```

That's rather independent of this PR, though :)